common/benchmark/src/AABBTreeBenchmark.cpp

```diff
@@ -75,8 +55,13 @@ namespace TrenchBroom {
         std::vector<AABB> trees(100);
         timeLambda([&world, &trees]() {
             for (auto& tree : trees) {
-                TreeBuilder builder(tree);
-                world->acceptAndRecurse(builder);
+                world->acceptLambda(kdl::overload {
```

Comment:
This needs to recurse over all of the node tree; as-is it won't insert anything into the AABB.

Reply:
Oh yeah, fixed.